internal/healthchecks/generic/generic.go

```diff
@@ -0,0 +1,198 @@
+package generic
```

Comment:
Not sure `generic` is super useful as a package name, especially when Go has generics and this code just does a `switch` on GVR.

Reply:
maybe `builtin`?

I assume the future-thinking bit of this package name is that there will eventually be some extension that enables users to provide custom health check logic, so we're reserving space for that under the healthchecks directory?

Reply:
Write for the code you have, not the code you wish you might have in the future :)

Reply:
Yeah, this is all in an internal directory, and we're in `v0`, so double-reason not to worry about future package name changes and moving of stuff around.

I'm also fine if we just put this directly in a `healthchecks` or similar package.

Reply:
done, moved into the healthchecks pakage and renamed to "builtin"